src/palladio/SOPAssign.cpp

```diff
@@ -68,6 +68,103 @@ RuleFileInfoUPtr getRuleFileInfo(const MainAttributes& ma, const ResolveMapSPtr&
 	return rfi;
 }
 
+enum class SpareParmType { FLT, BOOL, STR, FLT_ARRAY, BOOL_ARRAY, STR_ARRAY, FLT_ENUM, STR_ENUM, COLOR };
+using SpareParmTypeMap = std::map<std::wstring, SpareParmType>;
+
+SpareParmTypeMap getSpareParms(SOPAssign* node) {
```

Comment:
maybe rename to `getCGAAttributeTypes` or similar?

Reply:
renamed